src/screens/level.py

```diff
@@ -52,6 +53,12 @@ def __init__(
         self.display_surface = pygame.display.get_surface()
         self.switch_screen = switch
 
+        # cut scene
+        target_points = [(100, 100), (200, 200), (300, 100), (800, 900)]
+        speeds = [100, 150, 200]  # Different speeds for each segment
+        pauses = [0, 1, 0.5, 2]  # Pauses at each point in seconds
+        self.cut_scene_animation = SceneAnimation(target_points, speeds, pauses)
```

Comment:
This is more of a wording issue. 

A **cut scene** is a scene that was removed from a movie for various reasons, and is not in the final version.
A **cutscene**, on the other hand, is what you're trying to make possible, i.e. non-interactive sequences in a game

Reply:
I read your explaination on discord but i always invert 😅🙈